src/mem/cache/tags/base_set_assoc.hh

```diff
@@ -197,6 +209,14 @@ class BaseSetAssoc : public BaseTags
         // Increment tag counter
         stats.tagsInUse++;
 
+        // Notify partitioning policies of acquisition of ownership
+        for (auto partitioning_policy : partitioningPolicies){
```

Comment:
Nit: this works because we make copies of the pointers and then modify the thing being pointed to, but we should probably make it `auto &` to make it future proof.